src/pages/guides/using/contentlogs/contentlogs-setup.md

```diff
@@ -0,0 +1,177 @@
+
+  ![Go To End Of Stream](../../img/contentlogs/go_to_end.png "Go to End")
+
+- Verify Response status 204
+
+  ![Verify Response 204](../../img/contentlogs/verify_204.png "Verify 204")
+
+- Open event registration in new tab
+
+  ![Go to Event Registration](../../img/contentlogs/go_to_event_registration.png "Go to event registration")
+
+- Click on send sample event say Updated
+
+  ![Send Sample Event](../../img/contentlogs/send_sample_event_read.png "Send Sample event")
+
+- Go to Event Browser Tab. Click on `Go To Next Batch`
+
+  ![Go To Next Batch](../../img/contentlogs/go_to_next_batch.png "Go to Next Batch")
+
+- Verify Response Status 200 with content log of the updated sample event
+
+  ![Verify Resopnse 200](../../img/contentlogs/verify_200.png "Verify 200")
+  ![Check Payload](../../img/contentlogs/sample_event_received.png "Sample Event received in stream")
+
+
+## Programmatic Setup for Consumption of Content Logs Stream
+
+Once the above setup is complete and validated, we can continuously consume the stream of content logs from Adobe I/O events as and when operations are performed on assets.
+
+- Use the exposed event registration endpoint for the project.
+
+  ![Journaling API Endpoint](../../img/contentlogs/journaling_api_endpoint.png "Endpoint for Content Logs stream consumption")
+
+- Auth token needs to be passed in the request. Which can be generated using the cURL and all credentials under Oauth of the project.
+
+  ![Auth Token Endpoint](../../img/contentlogs/auth_token_endpoint.png "Endpoint for generating AuthToken")
+
+- Sample cURL for consuming the events stream
+
+  ![cURL for stream consumption](../../img/contentlogs/cURL_stream_consumption.png "Sample cURL for consumption")
+
+- Once we invoke the API, response headers has `rel="next"` link which can be used to fetch more events.
+
+  ![Next Link for fetching more events](../../img/contentlogs/next_link_headers.png "Fetching next link for more events")
+
+- Sample cURL using the next link from headers.
+
+  ![cURL for stream consumption using next link](../../img/contentlogs/cURL_stream_consumption_using_next_link.png "Sample cURL for consumption using next token")
+
+[Using Journaling API](https://developer.adobe.com/events/docs/guides/api/journaling-api) has more details on how to use the Journaling API to consume the stream of IO events.
+
+## Event Data Structure
+
+Content Log Events are structured in JSON format using the [CloudEvents](https://cloudevents.io/) spec
+
+*Example Event*
+
+````json
+{
+    "specversion": "1.0",
+    "id": "6b810378-0348-5931-8168-868910a71c9a",
+    "source": "urn:aio_provider_metadata:content_logs",
+    "type": "com.adobe.acp.contentlogs.read",
+    "datacontenttype": "application/json",
+    "time": "2025-06-30T17:37:44.744110791Z",
+    "eventid": "1abfc513-da4b-45ad-91ed-439ba7273b45",
+    "event_id": "1abfc513-da4b-45ad-91ed-439ba7273b45",
+    "recipient_client_id": "cd549c3e50314c8fb345f27e8be27dc2",
```

Comment:
I would recommend removing these fields from your sample payload as these will be eventually removed by end of 2025. See https://developer.adobe.com/events/docs/support/faq#why-do-i-see-duplicate-fields-for-recipient-client-id-and-event-id-in-the-delivered-payload